src/gui/transferlistmodel.h

```diff
@@ -30,15 +30,59 @@
 #ifndef TRANSFERLISTMODEL_H
 #define TRANSFERLISTMODEL_H
 
+#include "base/bittorrent/torrenthandle.h"
```

Comment:
you didn't remove this line in transferlistmodel.cpp

Reply:
And wrong order.

Reply:
now?